frontend/src/components/PaginationOffset.vue

```diff
@@ -276,11 +276,11 @@ input[type=number] {
     min-width: v-bind(minWidth);
 }
 
-div > button:hover {
+.navigation-container > div > button:hover {
```

Comment:
please replace to .navigation-container > button:hover
The div doesn't add any value